pycosep/community_separability.py

```diff
@@ -275,23 +278,22 @@ def _tsp_based_projection(pairwise_data, concorde_settings):
 
     # prepare TSP file
     with open(file_tsp_path, 'w') as file:
-        file.write("NAME : TSPS Concorde\n")
+        file.write(f"NAME : TSPS Concorde\n")
```

Comment:
Do you have the latest `main` in your branch? We do not need the formatting in cases where there is no placeholder (same for the other changes below).

Reply:
This was caused by editing the code from an outdated main. I have changed to the latest one.